src/global.css

```
@@ -3,7 +3,10 @@
 @import "tailwindcss";
 
 @import "@luminescent/ui";
-@import "@luminescent/ui/formatting";
+
+.markdown {
+  @import "@luminescent/ui/formatting";
```

Stylelint (17.5.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)